protocol/scripts/deploys/mainnet/v2/contract-addresses.ts

```diff
@@ -0,0 +1,429 @@
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                // No circuit breakers for Temple base strategy
+            },
+            GNOSIS_SAFE_STRATEGY1: {
+                ADDRESS: '0xbFD3c8A956AFB7a9754C951D03C9aDdA7EC5d638',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                UNDERLYING_GNOSIS_SAFE: '0x67ae2161449cf3c5528fea969beba3e4f3288c61',
+                CIRCUIT_BREAKERS: {
+                    DAI: '0x04f1A5b9BD82a5020C49975ceAd160E98d8B77Af',
+                    TEMPLE: '0xde79380FBd39e08150adAA5C6c9dE3146f53029e',
+                },
+            },
+            RAMOS_STRATEGY: {
+                ADDRESS: '0x96E303b6D807c0824E83f954784e2d6f3614f167',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                CIRCUIT_BREAKERS: {
+                    DAI: '0x87c470437282174b3f8368c7CF1Ac03bcAe57954',
+                    TEMPLE: '0x746a48E39dC57Ff14B872B8979E20efE5E5100B1',
+                },
+            },
+            TLC_STRATEGY: {
+                ADDRESS: '0x64f5219563e28EeBAAd91Ca8D31fa3b36621FD4f',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+            },
+        },
+        EXTERNAL: {
+            MAKER_DAO: {
+                DAI_TOKEN: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+                DAI_JOIN: '0x9759A6Ac90977b93B58547b4A71c78317f391A28',
+                POT: '0x197E90f9FAD81970bA7976f33CbD77088E5D7cf7',
+            },
+            BALANCER: {
+                VAULT: '0xBA12222222228d8Ba445958a75a0704d566BF2C8',
+                HELPERS: '0x5aDDCCa35b7A0D07C74063c48700C8590E87864E',
+                TOKEN: '0xba100000625a3754423978a60c9317c58a424e3D',
+            },
+        },
+    },
+    mainnet: {
+        CORE: {
+            TEMPLE_TOKEN: '0x470ebf5f030ed85fc1ed4c2d36b9dd02e77cf1b7',
+            CIRCUIT_BREAKER_PROXY: '',
+            // GNOSIS_SAFE_GUARD: '',
+            EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+            RESCUER_MSIG: '',
+        },
+        TREASURY_RESERVES_VAULT: {
+            ADDRESS: '',
+            D_USD_TOKEN: '',
+            D_TEMPLE_TOKEN: '',
+            TPI_ORACLE: '',
+        },
+        TEMPLE_LINE_OF_CREDIT: {
+            ADDRESS: '',
+            CIRCUIT_BREAKERS: {
+              DAI: '',
+              TEMPLE: '',
+            },
+            INTEREST_RATE_MODELS: {
+              LINEAR_WITH_KINK: '',
+            },
+        },
+        RAMOS: {
+            AUTOMATION_EOA: '0x67ae2161449cf3c5528fea969beba3e4f3288c61',
+            TEMPLE_DAI: {
+                ADDRESS: '0xC3133cB9e685ccc82C73FbE580eCeDC667B41917',
```

Comment:
I think these are unknown yet until we deploy - so should be blank:

* ADDRESS
* POOL_HELPER
* AURA_STAKING